### PR TITLE
update 2020/21 prices

### PR DESCRIPTION
**Pull request by vmartinv**
These are the prices missing for the last financial year, I got them from the HMRC website and from Schwab

### CODE REVIEW

File: calc.py [hunk 1]
```diff
@@ -24,10 +24,10 @@
 )
 
 # First year of tax year
-tax_year = 2019
-# Allowance is £12000 for 2019/20
+tax_year = 2020
+# Allowance for 2020/21
 # https://www.gov.uk/guidance/capital-gains-tax-rates-and-allowances#tax-free-allowances-for-capital-gains-tax
-capital_gain_allowance = 12000
+capital_gain_allowance = 12300
```
Comment (KapJI) [inline calc.py]: Let's create a map for this with year as a key and include allowances for different years.

Comment (vmartinv) [reply calc.py]: Added suggestions

File: calc.py [hunk 2]
```diff
@@ -34,10 +34,19 @@
...
 
 # First year of tax year
-tax_year = 2019
-# Allowance is £12000 for 2019/20
+tax_year = 2020
+# Allowances
 # https://www.gov.uk/guidance/capital-gains-tax-rates-and-allowances#tax-free-allowances-for-capital-gains-tax
-capital_gain_allowance = 12000
+capital_gain_allowances = {
+    2013: 12000,
```
Comment (KapJI) [inline calc.py]: I think they changed rules here, let's drop this one.
```suggestion
```

**Comment by KapJI** [comment]
Thanks!

**Pull request merged**